test/hotspot/jtreg/compiler/vectorization/cr8287517.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+* @test
+* @bug 8287517
+* @summary Test bug fix for JDK-8287517 related to fuzzer test failure in x86_64
+* @requires vm.compiler2.enabled
+* @run main/othervm -Xcomp -XX:CompileOnly=compiler/vectorization/cr8287517.test -XX:MaxVectorSize=8 compiler.vectorization.cr8287517
+*/
+
+package compiler.vectorization;
+
+public class cr8287517 {
```

Comment:
Please rename the test to something that describes the problem you are writing this test for instead of just using the bug number.

Reply:
@chhagedorn Thanks, changed test name to TestSmallVectorPopIndex.java.